bundles/org.smarthomej.transform.basicprofiles/README.md

```diff
@@ -42,6 +42,24 @@ Switch lightsStatus {
 }
 ```
 
+## Debounce (Counting) Profile
+
+This Profile counts and skips the number for State changes before it sends an update to the Item.
+It can be used to debounce Item States.
+The number of skipped changes can be defined by the user.
```

Comment:
```suggestion
This Profile counts and skips a user-defined number of State changes before it sends an update to the Item.
It can be used to debounce Item States.
```